backend/src/main/java/com/ambacam/rest/statusrequetes/StatusRequetesResource.java

```diff
@@ -0,0 +1,51 @@
+package com.ambacam.rest.statusrequetes;
+
+import java.util.List;
+
+import javax.validation.Valid;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.ambacam.model.StatusRequete;
+import com.ambacam.rest.ApiConstants;
+import com.ambacam.service.StatusRequeteService;
+
+@RestController
+@RequestMapping(ApiConstants.STATUS_REQUETE_COLLECTION)
+@Validated
+public class StatusRequetesResource {
+
+	private static final Logger log = LoggerFactory.getLogger(StatusRequetesResource.class);
+
+	@Autowired
+	private StatusRequeteService statusRequeteService;
+
+	@RequestMapping(method = RequestMethod.POST)
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.APPLICATION_JSON)
+	public StatusRequete create(@RequestBody @Valid StatusRequete create) {

```

Comment:
If in you change create->role in RolesResource please change here also create->statusRequete